apps/passport-server/src/services/generic-issuance/pipelines/LemonadePipeline.ts

```diff
@@ -738,18 +828,23 @@ export class LemonadePipeline implements BasePipeline {
           }
         }
 
-        const lemonadeEventId = ticketAtom.lemonadeEventId;
-
         this.pendingCheckIns.set(ticketAtom.id, {
           status: CheckinStatus.Pending,
           timestamp: Date.now()
         });
         try {
-          await this.api.checkinTicket(
-            this.definition.options.lemonadeApiKey,
-            lemonadeEventId,
-            // Is this the correct ticket ID?
-            ticketAtom.id
+          const credentials = {
+            oauthAudience: this.definition.options.oauthAudience,
+            oauthClientId: this.definition.options.oauthClientId,
+            oauthClientSecret: this.definition.options.oauthClientSecret,
+            oauthServerUrl: this.definition.options.oauthServerUrl
+          };
```

Comment:
here and elsewhere, it would be nice to define a type for whatever this `credentials` object is - `OAuthCredentials` ?

Reply:
There is a type for it - `LemonadeOAuthCredentials`.

Reply:
Though in this instance I didn't actually declare the variable as such. I think this is because I lifted the contents out of the call to `checkinUser` below.